### ISSUE TITLE
Gui error when using unicode symbols and setting a breakpoint

### ISSUE THREAD

**Issue by mschlechtinger** [jupyterlab/debugger#538]
Hi!

i just found a minor bug within the the debugger's GUI, which occurs if you have a unicode symbol within the code of a cell. 
e.g.,: `print("€")`. 
Whenever this happens, the GUI is not able to display the breakpoint and the highlighting within a cell, however it is still registered in the window to the right and the debugger works properly. The breakpoints for the other cells still display as usual.

this is how it looks like (Line 52):
![image](https://user-images.githubusercontent.com/14161426/92387602-2032dc00-f116-11ea-8589-05a95f6168e6.png)

This is how I'm expecting it to behave (Line 52):
![image](https://user-images.githubusercontent.com/14161426/92387342-a3076700-f115-11ea-8d65-44ae7bd967b8.png)

**Comment by jtpio** [comment]
Thanks @mschlechtinger.

I could reproduce the bug on Binder with the latest release with the following code snippet from the tutorial notebook:

```python
def add(a, b):
    res = a + b
    print('$€')
    return res
```

**Comment by jtpio** [comment]
Reporting here for visibility.

We looked into this with @JohanMabille, and it seems like the murmurhash implementation in JS gives a different value than the one used in `xeus-python`, for the same input.

Tested with:

```javascript
const hash = require('murmurhash-js');
const s = `def add(a, b):
    res = a + b
    print('€')
    return res`;

const h = hash.murmur2(s, 3339675911);
console.log(h);
```

Gives: `4275293905`

While `xeus-python` / `xtl` gives: `3616960136`

We tried with the `murmurhash2` python package: https://pypi.org/project/murmurhash2

![image](https://user-images.githubusercontent.com/591645/95189532-48256600-07ce-11eb-9feb-22f55b450a4e.png)

So it looks like the implementation in `murmurhash-js` might be falsy.
